ChangeLog.md

```diff
@@ -1,5 +1,10 @@
 Branch iOS SDK Change Log
 
+- v0.18.0
+  * Branch links opened via Air Drop now open correctly (GH-#699).
+    Testing notes:  Test this heavily on many iOS, and test opening links when app is quit,
+    and when app is in background.  Also test regular links from Notepad, etc.
```

Comment:
I would emphasize to test with the app in _foreground_ too (the faulty AirDrop scenario).